eng/versioning/version_client.txt

```diff
@@ -62,7 +62,9 @@ com.microsoft.azure:azure-spring-boot-test-core;2.3.2;2.3.3-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
-
+unreleased_com.azure:azure-core;dependency
```

Comment:
Missing dependency version. Should be `unreleased_com.azure:azure-core;1.6.0-beta.1`.

Same comment for the other unreleased tags.